gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagManagementDagActionStoreChangeMonitor.java

```diff
@@ -62,28 +59,18 @@ protected void handleDagAction(DagActionStore.DagAction dagAction, boolean isSta
     LaunchSubmissionMetricProxy launchSubmissionMetricProxy = isStartup ? ON_STARTUP : POST_STARTUP;
     try {
       // todo - add actions for other other type of dag actions
-      if (dagAction.getDagActionType().equals(DagActionStore.DagActionType.LAUNCH)) {
-        // If multi-active scheduler is NOT turned on we should not receive these type of events
-        if (!this.isMultiActiveSchedulerEnabled) {
```

Comment:
These dag actions should be handled in non-multi active scheduler mode also.

Reply:
at present launch events are only written to `dagActionStore` by the multi active scheduler mode though, otherwise passed directly from scheduler to orchestrator to `dagManager`. Is `dagProcEngine` ever to be enabled without multi-active scheduler mode? I don't think so, so we should keep this check. However, if we do want that option then we have to write launch events to `dagActionStore` instead of `Orchestrator` passing event to `DagManager` directly (or still allow it but have the change event do a no-op in non-multi active scheduler mode). Let's sync on this

Reply:
I foresee benefit in validating `DagProcEngine` correctness by running in single-active mode